compose.yml

```diff
@@ -0,0 +1,42 @@
+version: "3.9"
```

Comment:
What can we add to the compose file to make images build automatically?

Reply:
We have to add the "build" option with its context. For example, 
version: "3.9"
services:
   headnode2:
      build:
         context: https://raw.githubusercontent.com/MikhYur1991/devOPSp2/Docker/Dockerfile.headnode
      volumes:
         - h2mnt1:/opt/mount1
         - h2mnt2:/opt/mount2
         - h2logs:/usr/local/hadoop/current/logs
      networks:
         - hadoopnet
volumes:
   h2mnt1:
      name: h2mount1
   h2mnt2:
      name: h2mount2
   h2logs:
      name: headnode2logs
networks:
   hadoopnet:
      driver: bridge
      name: hadoopnet